modelmesh-monitoring/base/prometheus-operator/deployment.yaml

```diff
@@ -0,0 +1,56 @@
+kind: Deployment
```

Comment:
From a previous conversation with @anishasthana and @LaVLaS, we wanted to avoid having one operator depend on another. Hence the manual deployment

Reply:
Several things:
1. Prometheus operator, AFAICT, is part of openshift-monitoring which is part of the openshift platform, unless I misunderstood https://docs.openshift.com/container-platform/4.10/monitoring/monitoring-overview.html So that's a dependency on the platform rather than an external operator.
2. The manifest don't have the prometheus operator  CRDs, so it won't work on cluster that indeed do not have the prometheus-operator (bare k8s for example).
3. On clusters that do have the prometheus operator, there is the potential that the two instance of the operators fight themselves trying to control CR -> chaos ensues.

IMO, using the Openshift monitoring stack (and for k8s user, declaring a dependency on prometheus operator) is better.

Reply:
1. This part was brought up in previous discussions with @LaVLaS and @anishasthana. Eventually we want to have a unified monitoring solution for all components and then we can avoid having a prometheus operator deployed JUST for model monitoring (or any other individual component). But for the time being it was decided that it is okay to replicate what we have downstream for ODH as well. 
2. That can be added as a note in the readme, in my opinion. (that for bare k8s the operator install is needed as a prereq for model-monitoring to work)
3. This current setup has been deployed and tested downstream for at least a couple of releases, we have not observed this issue.

Reply:
3. Yeah I didn't mean it will happens, just that it's a potential source of problem.

Reply:
Explained and resolved offline with @VannTen and @anishasthana